File: Programmers/pro_12985_예상 대진표.py
def solution(n,a,b):
    answer = 1
    while(1):
        nextA = max(1,(a+1)//2)
        nextB = max(1,(b+1)//2)
        if(nextA==nextB):
            return answer
        a=nextA
        b=nextB
        answer+=1
    return answer
# 12 34 56 78 910....
# 1   2  3  4  5 
# (n+1) %2 = 현재 집단,다음 번호
# 현재 집단이 같으면 끝
# 아니면 다음 번호로 진행

File: Programmers/pro_12985_예상대진표.js
function solution(n, a, b) {
    [a, b] = [parseInt((a - 1) / 2), parseInt((b - 1) / 2)]
    let answer = 1;
    while (a != b) {
        answer += 1
        a = parseInt(a / 2)
        b = parseInt(b / 2)
    }
    return answer;
}